Automattic/jetpack
# VideoPress: fix issue when detecting autogenerated video track

Although this PR fixes the issue when the auto-generated chapter is detected, the changes introduced here process the response body when uploading a new track File on dotcom side.
It allows picking the data correctly to be served wherever required.

Fixes https://github.com/Automattic/jetpack/issues/27809

#### Changes proposed in this Pull Request:

* https://github.com/Automattic/jetpack/issues/27809

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the block editor
* Create/edit a video block instance
* Add chapters via the block description
* Confirm it works as expected when saving the post

before | after
------|------
<img width="833" alt="Screen Shot 2022-12-07 at 19 10 17" src="https://user-images.githubusercontent.com/77539/206273858-385bf57a-0130-472d-8051-af46042b1a68.png"> | <img width="836" alt="Screen Shot 2022-12-07 at 19 08 09" src="https://user-images.githubusercontent.com/77539/206273530-3f78f092-80f4-46cf-930c-ecad274f075c.png">
